src/dispatch/case/service.py

```diff
@@ -187,6 +189,24 @@ def create(*, db_session, case_in: CaseCreate, current_user: DispatchUser = None
         case_id=case.id,
     )
 
+    # add reporter
+    participant_flows.add_participant(
+        case_in.reporter.individual.email,
+        case,
+        db_session,
+        role=ParticipantRoleType.reporter,
+    )
+
+    # add observer
+    observer_email = case_in.observer.individual.email
+    if observer_email:
+        participant_flows.add_participant(
+            observer_email,
+            case,
+            db_session,
+            role=ParticipantRoleType.observer,
+        )
```

Comment:
I think observer(s) would make more sense to be resolved via engagement filters and added to the case after the case is created in the same way we do it for incidents rather than having one participant with observer role being added here. Also, who would that be on case creation?

Reply:
Removed observer role for now.